pages/juang961225/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>carrusel</title>
+    <link rel="stylesheet" href="../juang961225/style.css">
+</head>
+<body>
+<!--contenedor general-->
+    <main class="main">
+<!--titulo principal-->
+        <h2 class="main__title">word of warcraft class</h2>
+<!--contenedor de las cartas-->
+        <div class="main__container">
+<!--boton izquierdo de las cards-->
+        <button class="main__button">
+            <h2 class="main__container-button"><</h2>
+        </button>
+<!--primera carta-->
+        <div class="main__card">
+            <img class="main__img" src="../juang961225/pictures/paladin.jpg" alt="paladin">
+            <h2 class="main__card-title">paladin</h2>
+            <h3 class="main__content">first content</h3>
+            <h3 class="main__content">second content</h3>
+            <h3 class="main__content">third content</h3>
+            <div class=" main__container-button">
+                <button class="main__button-card">more ></button>
+            </div>
+        </div>
+<!--segunda carta-->
+        <div class="main__card">
+            <img class="main__img" src="../juang961225/pictures/death knight.jpg" alt="caballero de la muerte">
+            <h2 class="main__card-title">the title</h2>
+            <h3 class="main__content">first content</h3>
+            <h3 class="main__content">second content</h3>
+            <h3 class="main__content">third content</h3>
+            <div class=" main__container-button">
+                <button class="main__button-card">more ></button>
+            </div>
+        </div>
+<!--tercera carta-->
+        <div class="main__card">
+            <img class="main__img" src="./pictures/warrior.jpg" alt="warrior">
+            <h2 class="main__card-title">the title</h2>
+            <h3 class="main__content">first content</h3>
+            <h3 class="main__content">second content</h3>
+            <h3 class="main__content">third content</h3>
+            <div class=" main__container-button">
+                <button class="main__button-card">more ></button>
+            </div>
+        </div>
+<!--boton derecho de las cartas-->
+        <button class="main__button">
+            <h2 class="main__container-button">></h2>
+        </button>
+    </div>
+<!--botones inferiores-->
+    <button class="main__button-2"></button>
```

Comment:
I think you should reconsider wrap these buttons within a container and also they could be named better as "slide indicator" for example.

Reply:
I agree with Arley... these buttons do not look like buttons... a better name could be `dot` or `indicator`

Reply:
resolved